lib/utils/execution-state-manager.js

```diff
@@ -177,13 +178,55 @@ class ExecutionStateManager {
     this._browserToModuleMap.set(browserId, browserModuleList);
   }
 
+  /**
+   * Add module metadata mapped by moduleName to moduleMetadata Map.
+   *
+   * @param {string} moduleName
+   * @param {number} total - Total number of tests
+   * @param {number} passed - Number of passed tests
+   * @param {number} failed - Number of failed tests
+   * @param {number} runtime - Runtime to execute tests in module
+   * @param {Array<string>} failedTests - A list of failed test names
+   */
+  _injectModuleMetadata(moduleName, total, passed, failed, runtime, failedTests) {
+    this._moduleMetadata.set(
+      moduleName,
+      {
+        total: total,
+        passed: passed,
+        failed: failed,
+        runtime: runtime,
+        failedTests: failedTests
+      }
```

Comment:
This can be simplified to:
```{
        total,
        passed,
        failed,
        runtime,
        failedTests
      }```
if the key and the value has the same name, then js will automatically use the name of the variable as the key